mautic/mautic
# Feature gatedvideo builder

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | N
| New feature? | Y
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | N
| Deprecations? | N

#### Description:
This PR creates a new draggable Video slot in the Page Builder, where you're able to insert Gated Video as a slot into your pages. 

#### Steps to test this PR:
1. Apply PR
2. Edit a Landing Page in the builder, and see the new draggable slot in the list of slots on the top right.
3. Drag the slot over to your page, select it, and the form will appear on the right. Test using  YouTube, Vimeo, and MP4 as your url, and make sure the videos load on the landing page. Press play on the video and ensure that the selected form shows up, and that you're able to submit it.
